ballerina/client.bal

```diff
@@ -201,67 +200,121 @@ public isolated client class Client {
             requestHeaders, request);
         http:Response httpResponse = check self.amazonS3->put(requestURI, request);
         return handleHttpResponse(httpResponse);
     }
 
     # Deletes an object.
-    # 
+    #
     # + bucketName - The name of the bucket
     # + objectName - The name of the object
     # + versionId - The specific version of the object to delete, if versioning is enabled
     # + return - An error on failure or else `()`
     @display {label: "Delete Object"}
     remote isolated function deleteObject(@display {label: "Bucket Name"} string bucketName,
-                                    @display {label: "Object Name"} string objectName,
-                                    @display {label: "Object Version"} string? versionId = ())
+            @display {label: "Object Name"} string objectName,
+            @display {label: "Object Version"} string? versionId = ())
                                     returns @tainted error? {
         map<string> queryParamsMap = {};
         http:Request request = new;
         string queryParamsStr = "";
         string requestURI = string `/${bucketName}/${objectName}`;
 
         // Append query parameter(versionId).
-        if (versionId is string) {
+        if versionId is string {
             queryParamsStr = string `${queryParamsStr}?versionId=${versionId}`;
             queryParamsMap["versionId"] = versionId;
-        }    
+        }
         map<string> requestHeaders = setDefaultHeaders(self.amazonHost);
-        
+
         check generateSignature(self.accessKeyId, self.secretAccessKey, self.region, DELETE, requestURI,
             UNSIGNED_PAYLOAD, requestHeaders, request, queryParams = queryParamsMap);
         requestURI = string `${requestURI}${queryParamsStr}`;
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
-    }     
+    }
 
     # Deletes a bucket.
-    # 
+    #
     # + bucketName - The name of the bucket
     # + return - An error on failure or else `()`
     @display {label: "Delete Bucket"}
     remote isolated function deleteBucket(@display {label: "Bucket Name"} string bucketName) returns @tainted error? {
         http:Request request = new;
         string requestURI = string `/${bucketName}`;
         map<string> requestHeaders = setDefaultHeaders(self.amazonHost);
-        
+
         check generateSignature(self.accessKeyId, self.secretAccessKey, self.region, DELETE, requestURI,
             UNSIGNED_PAYLOAD, requestHeaders, request);
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
     }
+
+    # Generates a presigned URL for the object.
+    #
+    # + bucketName - The name of the bucket  
+    # + objectName - The name of the object  
+    # + expirationTime - The time until the presigned URL is valid  
```

Comment:
Can we mention the unit of time (e.g., seconds) since it's not conveyed via the param name either?